compiler/rustc_metadata/src/creader.rs

```diff
@@ -1032,6 +1032,43 @@ impl<'a, 'tcx> CrateLoader<'a, 'tcx> {
         }
     }
 
+    /// Inject the `compiler_builtins` crate if it is not already in the graph.
+    fn inject_compiler_builtins(&mut self, krate: &ast::Crate) {
+        // `compiler_builtins` does not get extern builtins, nor do `#![no_core]` crates
+        if attr::contains_name(&krate.attrs, sym::compiler_builtins)
+            || attr::contains_name(&krate.attrs, sym::no_core)
+        {
+            info!("`compiler_builtins` unneeded");
+            return;
+        }
+
+        // If a `#![compiler_builtins]` crate already exists, avoid injecting it twice. This is
+        // the common case since usually it appears as a dependency of `std` or `core`.
```

Comment:
of `std` and `alloc`. `core` doesn't depend on `compiler_builtins`. That is why we need this injection code in the first place.

Reply:
This just needs a comment update rather than behavior change, correct?

Reply:
Correct

Reply:
Updated the comment to "`std` or `alloc`".